### DISCUSSION TITLE
After setting the symbol of `x theme`, how to change back to the default set?

### DISCUSSION THREAD

**Discussion by MaxforCherubim** [x-cmd/x-cmd#71] [Q&A] [answered]
Now the theme I used is:

`[In]:   x theme current`
**`[Out]:  pl-svn/cyan-2`**

But my symbol is:

`[In]:  x theme feature cat`
**`[Out]:  theme:
  feature:
    default:
      symbol: china`**

How to change back to the original set of theme `pl-svn/cyan-2`?

**Comment by lunrenyi** [comment, marked as answer]
@MaxforCherubim  Are you trying to close or restore the configuration of the symbol?
```sh
x theme feature use symbol -t auto
```
<img width="411" alt="image" src="https://github.com/user-attachments/assets/45dffe9d-3bbb-42c8-8d64-ee2f2303fd8d">

**Reply by MaxforCherubim** [reply]
Yes, It work!
And I remembered I had somewhere command show  there is a "auto" in the symbol parameter, but I forget.
Anyway, thank you so much!
